### DISCUSSION TITLE
Is there a way to redirect routes with uppercase slug to lowercase slug ?

### DISCUSSION THREAD

**Discussion by lipoolock** [vercel/next.js#23798] [Help]
Hello

I would like to manage in highest level (next.config.js) a way to redirect all URL called with an uppercased slug to the same slug in lowercase

Example:
```
{
  source: /:MY-SLUG
  destination: /:my-slug
}
```

Is it doable ?

**Comment by barelyhuman** [comment]
I think you can use the vercel.json file if you are deploying this to vercel to do that , though being able to convert the variable to lowerCase is something you'll have to try.

**Reply by lipoolock** [reply]
Thanks for that advice

But unfortunately I'm not deploying on Vercel
Does it means that has to be handled on server config side only ?
